PyInstaller/hooks/hook-adios.py

```diff
@@ -0,0 +1,16 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2016, Axel Huebl
```

Comment:
Please use our standard Copyright header. You will be credited in the doc/CREDITS.rst file.